git-multimail/README.rst

```diff
@@ -505,6 +505,11 @@ multimailhook.maxCommitEmails
     mailbombing, for example on an initial push.  To disable commit
     emails limit, set this option to 0.  The default is 500.
 
+multimailhook.ignoreMergeRevisions
```

Comment:
Other variables to limit the scope of git-multimail include `multimailhook.refFilterExclusionRegex` so perhaps `excludeMergeRevisions` would be better name for consistency. What do you think?

Reply:
I've changed it and force-pushed to your branch.